### DISCUSSION TITLE
Drag&Drop function for merging videos function missing

### DISCUSSION THREAD

**Discussion by Flenkiha** [mifi/lossless-cut#1521] [General]
V3.39.0 offers a very useful function to change the order of files to be merged (thanks to GoPro this is necessary).
V3.53.0 offers NO function to do this.
Is this function really missing or just hidden somewhere?
If missing, why it was omitted? Any chance of adding this back soon?

**Comment by mifi** [comment]
You do it in the batch list

**Reply by Flenkiha** [reply]
Thank you! I have found it now! Keep up your good work! Cheers!
